Frontend/Gmfctn/src/app/modules/+auth/auth-routing.module.ts

```diff
@@ -0,0 +1,31 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+
+import { AuthComponent } from './auth.component';
+import { SignInComponent } from './components/sign-in/sign-in.component';
+
+
+const routes: Routes = [
+    {
+        path: '',
+        component: AuthComponent,
```

Comment:
Do we really need this component and this route?
As I see this component is only giving us a router-outlet, so it does nothing except providing a portal to the next level and as for me it could be skipped

Reply:
Fixed